modules/library/referencing/src/main/java/org/geotools/measure/Units.java

```diff
@@ -96,9 +96,24 @@ private Units() {
      */
     static {
         final UnitFormat format = SimpleUnitFormat.getInstance();
-        format.label(SEXAGESIMAL_DMS,      "D.MS");
-        format.label(DEGREE_MINUTE_SECOND, "DMS" );
-        format.label(PPM,                  "ppm" );
+        registerCustomUnits((SimpleUnitFormat) format);
+    }
+
+    /**
+     * Registers the labels and aliases for the custom units defined by Geotools.
+     * 
+     * @param format The UnitFormat in which the labels and aliases must be registered.
+     */
+    public static void registerCustomUnits(SimpleUnitFormat format) {
```

Comment:
This is the other key fix, provided by @dispiste, that is used to configure our built-in SimpleUnitFormat instances with common labels and alias.  Subsequent lists add to this list (for pixel and Kelvin) but do not change the approach.